jdaviz/components/viewer_data_select.vue

```diff
@@ -127,6 +127,10 @@ module.exports = {
       if (this.$props.viewer.reference === 'spectrum-2d-viewer') {
         multi_select = false
       }
+    } else if (this.$props.viewer.config === 'lcviz') {
+      if (this.$props.viewer.reference.startsWith('tpf')) {
```

Comment:
nope, suggestions welcomed!

Reply:
Some words for thought: in Kepler and TESS we call the usual spatial image data product a "postage stamp," for stars in the KIC/TIC. In TESS you can also use `tesscut` to cutout an analogous small spatial region for the same purpose. I wouldn't want to call the viewer `stamp` since not all TPF cubes are postage stamps. I like `aperture`, but I'd be cautious about that, only because we might make an aperture-related plugin, and we wouldn't want to confuse the viewer and plugin names. `image` is the least ambiguous – this is one of the very few types of images we expect to see in lcviz.

Reply:
ok, I'll remove this from this PR at least until a decision is made in lcviz.  I think image is probably fine as well and we can always change it to be more specific if needed.

Reply:
This isn't removed yet but I got a ping to review. What is going on? Hope you can clarify. Thanks.

Reply:
ah I removed it from the other place (title case), I still need it for the if-statements but will update with the new label suggested by @bmorris3 that has been merged for lcviz.  Sorry about that!

I think we still need this here until we prioritize a significant refactor to allow downstream overriding.